test/db/cmd/cmd_help

```diff
@@ -58,9 +58,9 @@ CMDS=<<EOF
 # ?X
 # ?l
 # ?t is used for get the execution time, thus shall be ignored.
-?!
-?+
-?-
+#?!
+#?+
+#?-
```

Comment:
why commenting?

Reply:
These commands print help when wrong arguments (or no arguments) are passed instead of just printing the `core->num->value`. We have `?v` command group for that.

Reply:
These checks were there to test whether they print the `core->num->value` or not